[Qt] Readme changes from February 22nd to November 12th (#1350)
Esri/arcgis-runtime-samples-qt: ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/README.md

## How to use the sample

Tap "Start" to parse the NMEA sentences into a simulated location data source, and initiate the location display. Tap "Recenter" to recenter the location display. Tap "Reset" to reset the location display.

## How it works

> 1. Load an NMEA string and parse the sentences into a series of locations. ‼️ **(Desktop platform need more details about using GPS dongle)**
> 2. Create an `AGSSimulatedLocationDataSource` and initialize it with the mock locations. ‼️ **(We can also directly parse the raw NMEA strings, and use `AGSNMEALocationDataSource` instead. Don't forget ti adjust the wordings)**
> 3. Set it to the location display's data source.

Reviewer: ```suggestion
1. Load an NMEA string and parse the sentences into a series of locations.
2. Create an `NmeaLocationDataSource` and initialize it with the mock locations.
3. Set it to the location display's data source.
```
Make sure to translate class names from different SDKs to Qt. Here `AGSSimulatedLocationDataSource` is an iOS class. However this sample uses `NmeaLocationDataSource` regardless.

4. Start the location display to begin receiving location and satellite updates.

## Relevant API

> * AGSLocation
> * AGSLocationDisplay
> * AGSNMEALocationDataSource
> * AGSNMEASatelliteInfo
> * AGSSimulatedLocationDataSource ‼️ remove this if you don't use a simulated data source

Reviewer: ```suggestion
* Location
* LocationDisplay
* NmeaLocationDataSource
```
We don't have an equivalent `AGSNMEASatelliteInfo` class yet, and this sample doesn't use a simulated location data source.

## About the data

A list of NMEA sentences is used to initialize a `SimulatedNMEADataSource` object. This simulated data source provides NMEA data periodically, and allows the sample to be used on devices without a GPS dongle that produces NMEA data.

> he route taken in this sample features a [2-minute driving trip around Redlands, CA](https://arcgis.com/home/item.html?id=d5bad9f4fee9483791e405880fb466da).

Reviewer: ```suggestion
The route taken in this sample features a [two-minute driving trip around Redlands, CA](https://arcgis.com/home/item.html?id=d5bad9f4fee9483791e405880fb466da).
```
Grammatically speaking, anything under 10 should be written as a word.
